### ISSUE TITLE
stripping.md needs diagrams

### ISSUE THREAD

**Issue by seneubert** [lhcb/first-analysis-steps#103]
This actually holds for all the theory parts: We need more illustrations to quickly convey the main concepts.

**Comment by alexpearce** [comment]
yep :+1: showing the general processing flow would be cool, showing where data and MC merge.

**Comment by apuignav** [comment]
Agree. Didn't have time to prepare them.

On Thu, Jun 4, 2015 at 12:42 PM, Alex Pearce notifications@github.com
wrote:

> yep [image: :+1:] showing the general processing flow would be cool,
> showing where data and MC merge.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/lhcb/first-analysis-steps/issues/103#issuecomment-108835744
> .

## 

Dr. Albert Puig Navarro
Laboratoire de Physique des Hautes Energies
Ecole Polytechnique Fédérale de Lausanne (EPFL)
BSP 614.4 (Cubotron UNIL) CH-1015 Lausanne
EPFL Phone: 021 6939808
CERN Phone: 72518

**Comment by laurentdufour** [comment]
Something like http://www.nikhef.nl/~laurentd/Overview_cut.pdf? I want to make the distinction still between offline & online (idea from Maarten) and made green what we kind-of-understand after the tutorials covered. Anything to add?

**Comment by betatim** [comment]
This looks pretty cool! Can you open a PR with that file in it please? If there is a source/editable version of the file it would be good to attach that as well.

**Comment by apuignav** [comment]
It's quite nice! A small comment on it -> "Collisions".

Looking forward to the PR.

**Comment by alexpearce** [comment]
Is this now covered by the diagram in the [dataflow lesson](https://lhcb.github.io/first-analysis-steps/07-dataflow.html) @seneubert?
